src/organisms/MainNavBar.test.jsx

```diff
@@ -50,3 +64,15 @@ test('Right Dropdown menu click invokes SetState function to remove "show"', ()
   dropdownMenu.simulate('click', { preventDefault() {} });
   expect(mockSetState).toHaveBeenCalledWith({ leftMenu: '', rightMenu: '' });
 });
+test('Right Dropdown menu click invokes SetState function to remove "show"', () => {
+  const wrapper = setup('', 'show');
+  const dropdownMenu = findByTestAttr(wrapper, 'rightdropdownMenu');
+  dropdownMenu.simulate('keydown', { keyCode: 13 });
```

Comment:
I wouldn't get this granular with a test for a key code. I'd throw this test out probably

Reply:
I'll just leave the test there for now, doesn't hurt